test/halmos/HalmosTest.sol

```diff
@@ -47,6 +51,23 @@ contract HalmosTest is SymTest, Test {
         morpho.enableLltv(lltv);
         morpho.createMarket(marketParams);
         vm.stopPrank();
+
+        // for flashLoan
+        otherToken = new ERC20Mock();
+        flashBorrower = new FlashBorrowerMock(morpho);
+
+        // Enable symbolic storage
+        svm.enableSymbolicStorage(address(morpho));
+        svm.enableSymbolicStorage(address(loanToken));
+        svm.enableSymbolicStorage(address(collateralToken));
+        svm.enableSymbolicStorage(address(oracle));
+        svm.enableSymbolicStorage(address(irm));
+        svm.enableSymbolicStorage(address(otherToken));
+        svm.enableSymbolicStorage(address(flashBorrower));
```

Comment:
svm.enableSymbolicStorage() is now used in place of --symbolic-storage.

Reply:
Right, it makes sense now to enable for all possible contracts !

Reply:
you may also want to include `this` address, especially if you add some callback functions in the test contract:
```
svm.enableSymbolicStorage(address(this));
```

Reply:
added in https://github.com/morpho-org/morpho-blue/pull/692/commits/a80998eba45966881cb4ab645395b38f710f429f